src/services/status.js

```diff
@@ -4,29 +4,41 @@ import config from '../config';
 import http from '../utils/http';
 
 import * as icons from '../constants/icons';
-import * as statuses from '../constants/statuses';
+import * as statusmessage from '../constants/statuses';
 import * as outage from '../constants/enums/outage';
 
 /**
- * Get the latest status of the services.
  *
- * @returns {Promise}
+ * @param {*} tokenId
+ * @returns {data}
  */
+
 export async function fetchServiceStatuses() {
   const { endpoints } = config.api;
   const { data } = await http.get(endpoints.status);
 
   return data;
 }
 
+export async function fetchLogs() {
+  const { endpoints } = config.api;
+  const { data } = await http.get(endpoints.statusLogs);
+
+  return data;
+}
+
 /**
  * Check if a service is up by it's status.
  *
- * @param {Object} service
+ * @param {Object} status
  * @returns {Boolean}
  */
-export function isUp(service) {
-  return service.status === statuses.STATUS_UP;
+export function isUp(status) {
```

Comment:
Always place one return outside block inside function
```js
export function isUp(status) {
  if (status.name === statusmessage.STATUS_UP) {
    return 1;
  }

  return 0;
}
```
Can we uses `true` `false` instead of `1` `0`?

Reply:
Or we can just do 
```js
export function isUp(status) {
  return (status.name === statusmessage.STATUS_UP);
}
```